src/SlashCommands.js

```diff
@@ -606,6 +607,26 @@ export const CommandMap = {
         },
     }),
 
+    addwidget: new Command({
+        name: 'addwidget',
+        args: '<url>',
+        description: _td('Adds a custom widget by URL to the room'),
+        runFn: function(roomId, args) {
+            if (!args || (!args.startsWith("https://") && !args.startsWith("http://"))) {
+                return reject(_t("Please supply a https:// or http:// widget URL"));
+            }
+            if (WidgetUtils.canUserModifyWidgets(roomId)) {
+                const userId = MatrixClientPeg.get().getUserId();
+                const nowMs = (new Date()).getTime();
+                const widgetId = encodeURIComponent(`${roomId}_${userId}_${nowMs}`);
+                return success(WidgetUtils.setRoomWidget(
+                    roomId, widgetId, "customwidget", args, "Custom Widget", {}));
```

Comment:
It looks like widget types are meant to be namespaced, such as `m.stickerpicker`. How about:
```suggestion
                    roomId, widgetId, "im.vector.customwidget", args, "Custom Widget", {}));
```

Reply:
Technically it should be m.customwidget - will fix